os/Kconfig.debug

```diff
@@ -134,6 +134,31 @@ config DEBUG_BINMGR_INFO
 
 endif # DEBUG_BINMGR
 
+config DEBUG_BINARY_COMPRESSION
```

Comment:
same as previous. Let's move this before binmgr because we list them by Alphabet order.

Reply:
Done.